src/i18n_check/check/all_checks.py

```diff
@@ -2,15 +2,41 @@
 """
 Runs all i18n checks for the project.
 
-Usage
------
-python3 src/i18n_check/checks/run_i18n_checks.py
+Examples
+--------
+Run the following script in terminal:
+
+>>> python3 src/i18n_check/checks/run_i18n_checks.py
```

Comment:
Really appreciate you checking the docs and using the correct style here, @Kal-Andor! 😊